features/scripts/export_ios_app.sh

```diff
@@ -2,33 +2,44 @@
 
 set -o errexit
 
-cd features/fixtures/ios
-
-echo "--- iOSTestApp: xcodebuild archive"
-
-#
-# Using CLANG_ENABLE_MODULES=NO to surface build errors
-# https://github.com/bugsnag/bugsnag-cocoa/pull/1284
-#
-
-xcrun xcodebuild \
-  -scheme iOSTestApp \
-  -workspace iOSTestApp.xcworkspace \
-  -destination generic/platform=iOS \
-  -configuration Debug \
-  -archivePath archive/iosTestApp.xcarchive \
-  -allowProvisioningUpdates \
-  -quiet \
-  archive \
-  CLANG_ENABLE_MODULES=NO \
-  GCC_PREPROCESSOR_DEFINITIONS='$(inherited) BSG_LOG_LEVEL=BSG_LOGLEVEL_DEBUG BSG_KSLOG_ENABLED=1'
-
-echo "--- iOSTestApp: xcodebuild -exportArchive"
-
-xcrun xcodebuild \
-  -exportArchive \
-  -archivePath archive/iosTestApp.xcarchive \
-  -destination generic/platform=iOS \
-  -exportPath output/ \
-  -quiet \
-  -exportOptionsPlist exportOptions.plist
+# "Release" or "Debug" must be specified
+if [ "$1" != "Release" ] && [ "$1" != "Debug" ]; then
+  echo "Usage: $0 [release|debug]"
+  exit 1
+fi
+
+pushd features/fixtures/ios
+
+  echo "--- iOSTestApp: xcodebuild archive"
+
+  #
+  # Using CLANG_ENABLE_MODULES=NO to surface build errors
+  # https://github.com/bugsnag/bugsnag-cocoa/pull/1284
+  #
+
+  xcrun xcodebuild \
+    -scheme iOSTestApp \
+    -workspace iOSTestApp.xcworkspace \
+    -destination generic/platform=iOS \
+    -configuration $1 \
```

Comment:
Could `$1` be grabbed at the top of the script and given a name? I've seen too many horror stories when scripts get modified ;-)

Reply:
Not a bad idea at all - I'll just change it in my next PR to avoid a conflict.